test/DelegationTokenInternalTest.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {Test, console} from "../lib/forge-std/src/Test.sol";
+import {DelegationToken} from "../src/DelegationToken.sol";
+
+contract DelegationTokenInternalTest is Test, DelegationToken {
+    uint256 internal constant MAX_TEST_AMOUNT = 1e28;
+
+    function getInitializableStorage() internal pure returns (InitializableStorage storage $) {
```

Comment:
In favor of avoiding named return values in declaration as per previous observation.

Reply:
We only need this function because `_getInitializableStorage` is external and not public. So I think here we should keep the same syntax as on the source file, so that it's clear that it's the same

Reply:
Maybe we should change it in the source file as well...

Reply:
Forgot to mention, consider this comment as optional.